src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -113,6 +124,18 @@ export const TranscriptsListItemInfo = (
 
   const mainStyles = classNames(transcriptsListStyles.row, styles.listItemInfo);
   const midStyles = classNames(transcriptsListStyles.middle, styles.middle);
+  const transcriptCCDS = transcript.external_references.find((x) =>
+    x.accession_id.match('CCDS')
+  );
+  const transcriptMetaData =
```

Comment:
const hasRelevantMetadata = ['gencode_basic', 'tsl', 'appris'].some(key => key in transcript.metadata);

Reply:
This wont work because the key will exist but wont have data (data null). see example:

```
        {
          "stable_id": "ENST00000665585.1",
          "metadata": {
            "gencode_basic": null,
            "appris": null,
            "tsl": null
          }
        },
```

Reply:
Ok, `transcript.metadata[key]` then 😄

Reply:
Thanks for this, i have made the changes. can you please review.